uiowa/uiowa
# starting the 2plus2 stuff

This adds the banner and pre-title for a 2plus2 major.

# How to test

composer install.
Sync admissions.
`yarn workspace admissions_core sass-compile && ddev drush @admissions.local cr`
create a community college taxonomy term.
create a piece of `Major (2 plus 2)` content, using your Community college and any areas of study that you would like.
Add a featured image to this piece of content.
Verify that the pre-title has a yellow background and black text, and that the title has white text over top of the banner picture.
Verify that your community college is brought in correctly into the title
